app/.eslintrc.json

```diff
@@ -19,6 +19,7 @@
     "react/prop-types": "off",
     "no-console": "warn",
     "no-warning-comments": "warn",
+    "camelcase": "off",
```

Comment:
This seems like it can be removed (I removed it and it didn't trigger any error).